client/lib/use-in-view/index.ts

```diff
@@ -1,39 +1,49 @@
 import { RefObject, useEffect, useRef } from 'react';
 
-export function useInView< T extends Element >( oneTimeCallback: () => void ): RefObject< T > {
+type OptionalProps = {
+	[ key: string ]: unknown;
+};
+
+export function useInView< T extends Element >(
+	oneTimeCallback: () => void,
+	optionalProps?: OptionalProps
```

Comment:
It might be better to call it `dependencies` and the default value is an empty array. In our cases, we could

```
useInView( trackScrollToEnd, [ category.selection ] );
```

What do you think?

Reply:
That would be great! Thanks for suggestion